.gitignore

```diff
@@ -112,6 +112,9 @@ dist
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
 
+# Parcel cache
+.parcel-cache/
```

Comment:
`.npmignore` must be changed accordingly.

Reply:
Will do